static/app/views/replays/detail/accessibility/details/index.tsx

```diff
@@ -0,0 +1,91 @@
+import {Fragment, MouseEvent} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Stacked from 'sentry/components/replays/breadcrumbs/stacked';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {HydratedA11yFrame} from 'sentry/utils/replays/hydrateA11yFrame';
+import {useResizableDrawer} from 'sentry/utils/useResizableDrawer';
+import AccessibilityDetailsContent from 'sentry/views/replays/detail/accessibility/details/content';
+import SplitDivider from 'sentry/views/replays/detail/layout/splitDivider';
+
+type Props = {
+  item: null | HydratedA11yFrame;
+  onClose: () => void;
+  projectId: undefined | string;
+  startTimestampMs: number;
+} & Omit<ReturnType<typeof useResizableDrawer>, 'size'>;
+
+function AccessibilityDetails({
+  isHeld,
+  item,
+  onClose,
+  onDoubleClick,
+  onMouseDown,
+  projectId,
+  startTimestampMs,
+}: Props) {
+  if (!item || !projectId) {
+    return null;
```

Comment:
When does this co ndition happen? Should there be some sort of message?

Reply:
not sure. I'll dig into the types a little more to findout.